### PR TITLE
Update th.m3u

### PR DESCRIPTION
**Pull request by Soponwit19997**

### CODE REVIEW

**Review by Soponwit19997 — COMMENTED** [review]
Update th.m3u 01/09/2023

File: streams/th.m3u [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
+#EXTINF:-1 tvg-id="CatoonClub.th",Cartoon Club [Not 24/7]
+https://edge1.bkk-3bb.co.th:9443/cartoonLiveApp/cartoonLiveApp.stream/chunklist_w859044783.m3u8
 #EXTINF:-1 tvg-id="Channel3.th",Channel 3 (720p)
 https://cdn6.goprimetime.info/feed/202306140918/ch3hd3/index.m3u8
 #EXTINF:-1 tvg-id="Channel5.th",Channel 5 (1080p)
 https://639bc5877c5fe.streamlock.net/tv5hdlive/tv5hdlive/playlist.m3u8
-#EXTINF:-1 tvg-id="Channel5.th",Channel 5 (480p)
-https://tc-live1.sanook.com/live/22302_ch5.m3u8
 #EXTINF:-1 tvg-id="Channel7.th",Channel 7 (1080p)
-http://streaming-hwc.ch7.com/livech7hd/HD_1080p.m3u8
```
Comment (freearhey) [inline streams/th.m3u:19]: Working
image

File: streams/th.m3u [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
-#EXTINF:-1 tvg-id="Channel7.th",Channel 7 (720p)
-http://streaming-hwc.ch7.com/livech7hd/HD_720p.m3u8
```
Comment (freearhey) [inline streams/th.m3u:21]: Working
image

File: streams/th.m3u [hunk 3]
```diff
@@ -1,28 +1,24 @@
...
+https://live-cdn-hwc.ch7.com/livech7hd/HD_1080p.m3u8?vhost=streaming-hwc.ch7.com
```
Comment (freearhey) [inline streams/th.m3u:15]: Offline
image

**Review by freearhey — CHANGES_REQUESTED** [review]

**Pull request closed without merging**